drivers/tee/optee/optee_bench.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2016, Linaro Limited
+ *
+ * This software is licensed under the terms of the GNU General Public
+ * License version 2, as published by the Free Software Foundation, and
+ * may be copied, distributed, and modified under those terms.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
+
+#ifndef _OPTEE_BENCH_H
+#define _OPTEE_BENCH_H
+
+#include <linux/tee_drv.h>
+
+#define OPTEE_BENCH_DIVIDER 		64
+/* max amount of timestamps */
+#define OPTEE_BENCH_MAX_STAMPS	32
```

Comment:
I'll move this definition (`OPTEE_BENCH_MAX_STAMPS`) to common.mk in build, it's should be easy to change it in all sublayers